node/src/blockchain/blockchain_interface/blockchain_interface_web3/mod.rs

```diff
@@ -1768,8 +1465,18 @@ mod tests {
         let mut sign_transaction_params = sign_transaction_params_arc.lock().unwrap();
         let (transaction_params, _, secret) = sign_transaction_params.remove(0);
         assert!(sign_transaction_params.is_empty());
-        assert!(transaction_params.gas >= U256::from(not_under_this_value));
-        assert!(transaction_params.gas <= U256::from(not_above_this_value));
+        assert!(
+            transaction_params.gas >= U256::from(not_under_this_value),
+            "actual gas limit {} isn't above {}",
```

Comment:
This message is a little misleading; it makes things look like the comparison is > rather than >=. You could say "isn't above or equal to {}", or you could say "is below {}".